src/lib/intl.d.ts

```diff
@@ -103,7 +108,7 @@ declare module Intl {
     }
 
     interface DateTimeFormat {
-        format(date: number): string;
+        format(date?: Date | number): string;

```

Comment:
thanks. I did a quick check on different browsers and seems they all support all values as you mentioned.